packages/types/src/scope.ts

```diff
@@ -9,45 +9,65 @@ import { User } from './user';
  */
 export interface Scope {
   /** Add new event processor that will be called after {@link applyToEvent}. */
-  addEventProcessor(callback: EventProcessor): Scope;
+  addEventProcessor(callback: EventProcessor): this;
 
   /**
    * Updates user context information for future events.
-   * @param user User context object to be set in the current context.
+   * @param user User context object to be set in the current context. If null or undefined user will be unset.
    */
-  setUser(user: User): Scope;
+  setUser(user?: User): this;
 
   /**
-   * Updates tags context information for future events.
+   * Set an object that will be sent as tags data with the event.
    * @param tags Tags context object to merge into current context.
    */
-  setTag(key: string, value: string): Scope;
+  setTags(tags?: { [key: string]: string }): this;
```

Comment:
It's partially true. If you pass `setTags({})` it will send old tags, as nothing will get merged. There should also be mention about unsetting.